src/server/handlers/github.js

```diff
@@ -321,6 +322,20 @@ const getRequest = (owner, name, token, secret) => {
   };
 };
 
+
+export const hasRepoChanged = async (repositoryUrl, last_updated_at, token) => {
+  if (last_updated_at === undefined || !last_updated_at) {
+    throw new Error('`last_updated_at` must be given.');
+  }
+  const since = moment(last_updated_at).toISOString();
+  const { owner, name } = parseGitHubRepoUrl(repositoryUrl);
+  const uri = `/repos/${owner}/${name}/commits?since=${since}`;
+  const response = await requestGitHub.get(uri, { token, json: true });
+  await checkGitHubStatus(response);
```

Comment:
Yup, will work on something clever for handling 403 & X-RateLimit-Remaining / -Reset  response headers.
This associated with GH authentication to escape 60 req/h limit